geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -229,27 +233,12 @@ integer (+/-&nbsp;9223372036854775807) for CURSOR.
 | server       | redis_version<br/>redis_mode (always returns "cluster" because <%=vars.product_name%> for Redis always runs in cluster mode.)<br/>tcp_port<br/>uptime_in_seconds<br/>uptime_in_days |
 | stats        | total_commands_processed<br/>instantaneous_ops_per_sec<br/>total_net_input_bytes<br/>instantaneous_input_kbps<br/>total_connections_received<br/>keyspace_hits<br/>keyspace_misses<br/>evicted_keys (always returns 0)<br/>rejected_connections (always returns 0)<br/>pubsub_channels<br/>pubsub_patterns|
 
-## <a id="advantages-over-redis"></a>Advantages of <%=vars.product_name%> over Redis
-
-<%=vars.product_name%>’s primary advantage is its **scalability**. While the Redis server is single threaded, <%=vars.product_name%> supports high concurrency. Many Redis clients can execute commands on the <%=vars.product_name%> cluster simultaneously.
-
-<%=vars.product_name%>'s architecture and management features help detect and resolve **network partitioning** problems without explicit management on the part of the Redis client.
-
-<%=vars.product_name%> for Redis partitions data across multiple servers and keeps replicated data up to date _synchronously_, whereas Redis uses asynchronous replication.
-This provides a higher level of data consistency within the cluster.
-
 ## <a id="expiration-accuracy"></a>Expiration Accuracy
 
 Keys are expired in two ways, actively and passively:
 
--   With active expiration, expiration is evaluated whenever a key is accessed. If the key is due to expire, it is deleted. Active expiration is accurate to the millisecond.
--   With passive expiration, keys are evaluated every three minutes. If they are due to expire, they are deleted.  Passive expiration is accurate to the second.
-
-## <a id="high-availability-model"></a>High Availability Model
-
-Data is stored in a single partitioned region that by default has one redundant copy.
-In practice this means that the cluster can tolerate the loss of a single server without the loss of
-data.
+-   With active expiration, expiration is evaluated **whenever a key is accessed**. If the key is due to expire, it is deleted. Active expiration is accurate to the millisecond.
+-   With passive expiration, keys are evaluated **every three minutes**. If they are due to expire, they are deleted.  Passive expiration is accurate to the second.
```

Comment:
I'm not certain about the "Passive expiration is accurate to the second" here. What does this actually mean? If we're only checking for passive expiration every three minutes, it seems like we're only accurate to within three minutes? Maybe this would be better removed?

Reply:
Agreed.  Removed and updated to reflect feedback.